requirements.txt

```diff
@@ -1,10 +1,11 @@
 beautifulsoup4==4.13.3
+bs4==0.0.1
```

Comment:
This isn't necessary - maybe it got added by an automated tool? `bs4` is a dummy package for `beautifulsoup4` and you're supposed to install the latter: https://pypi.org/project/bs4/

Reply:
Thanks!